Gopkg.lock

```diff
@@ -328,6 +328,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "ff4a4b53ef454edf0ae2ec3bc497baf74d561af328faa123eb472d87954a906c"
+  inputs-digest = "b19769641bf117041022e3de0c490313047f3b7aaef54e00a7354573fff9b738"
```

Comment:
No need to change this?

Reply:
It is updated automatically after I ran `update_vendor.sh`. Do I need to revert?

Reply:
yes